tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -17,9 +17,14 @@
 @mock.patch("cachito.workers.pkg_managers.general_js.generate_and_write_npmrc_file")
 @mock.patch("cachito.workers.pkg_managers.general_js.RequestBundleDir")
 @mock.patch("cachito.workers.pkg_managers.general_js.run_cmd")
-def test_download_dependencies(mock_run_cmd, mock_rbd, mock_gawnf, mock_td):
+@mock.patch("shutil.move")
+def test_download_dependencies(mock_move, mock_run_cmd, mock_rbd, mock_gawnf, mock_td):
     mock_td.return_value.__enter__.return_value = "/tmp/cachito-agfdsk"
-
+    mock_run_cmd.return_value = (
+        "angular-devkit-architect-0.803.26.tgz angular-animations-8.2.14.tgz "
+        "rxjs-6.5.5-external-78032157f5c1655436829017bbda787565b48c30.tgz"
+    )
+    mock_move.return_value = "File moved"
```

Comment:
Could you remove this?

Reply:
Oops I had added that just while testing. Removing!!